nexus/options.go

```diff
@@ -26,6 +26,8 @@ type StartOperationOptions struct {
 	// Request ID that may be used by the server handler to dedupe a start request.
 	// By default a v4 UUID will be generated by the client.
 	RequestID string
+	// Links contain information about the caller event that made the Nexus call.
```

Comment:
I'm wondering if we need to support multiple links in this direction, I was thinking a slice was only required in the response. I will think about it and bring it up for discussion.